setup.py

```diff
@@ -46,7 +53,10 @@ def parse_requirements_file(path):
                 # installed
                 continue
             line = line.strip()
-            reqs.append(line)
+            if line[0] == "-":
```

Comment:
Let's remove it, since it's not used anymore. We can add later if necessary, but I also try to avoid installing directly from github unless there is no other choice.